src/components/atoms/BigStoryCardAutoSlider/index.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import PropTypes from "prop-types";
+import get from "lodash/get";
+import { Link } from "@quintype/components";
+import ResponsiveImageWithFallback from "../ResponsiveImageWithFallback";
+import Contributor from "../Contributor/index";
+import { getStoryData, generateImageSources } from "../../utils/utils";
+import "./bigStoryCardAutoSlider.m.css";
+import Headline from "../Headline";
+
+const BigStoryCardAutoSlider = ({ story, cardWithImageZoom = true, className = "" }) => {
+  const storyData = getStoryData(story);
+
+  if (!storyData.headline || !story.url) {
+    return null;
+  }
+
+  const contributorName = get(story, ["authors", 0, "name"], "");
+  const contributorRole = get(story, ["authors", 0, "contributor-role", "name"], "");
+  const externalLink = get(story, ["metadata", "reference-url"]);
+
+  return (
+    <Link
+      className={`${"base"} ${className} ${cardWithImageZoom ? "card-with-image-zoom" : ""}`}
+      aria-label={`${"Read full story: "} ${storyData.headline}`}
+      href={externalLink || story.url}
+      externalLink={externalLink}
+    >
+      <ResponsiveImageWithFallback
+        styleName="slider-image-wrapper"
+        slug={storyData.imageS3Key}
+        metadata={storyData.imageMetadata}
+        alt={storyData.imageCaption}
+        imgParams={{ auto: ["format", "compress"] }}
+        sources={generateImageSources(
+          { aspectRatio: [2, 1], screenWidthCoverage: 1 },
+          { aspectRatio: [2, 1], screenWidthCoverage: 1 }
+        )}
+      >
+        <div styleName="text-area">
+          <div styleName="text-wrapper">
+            {contributorName && <Contributor name={contributorName} type={contributorRole} styleName="contributor" />}
+            <Headline text={storyData.headline} headerType={9} headerLevel={3} styleName="headline highlight" />
```

Comment:
can we give this a className instead of styleName, I have followed the className everywhere